notes/zrq/20220121-02-arcus-testing.txt

```diff
@@ -0,0 +1,253 @@
+    >                   'status': 'SLOW',
+    >                   'msg': '',
+    >                   'valid': 'TRUE'
+    >                   }
+    >               }
+    >           ",
+    >       "stdout_lines": [
+    >           "Test completed after: 9289.35 seconds",
+    >           "...."
+    >           ]
+    >       }
+    >
+    >   PLAY RECAP ****************************************************************************************
+    >   localhost   : ok=9    changed=6    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+    >
+    >   /
+    >
+    >   real	196m32.410s
+    >   user	41m58.594s
+    >   sys	7m21.803s
+
+
+# -----------------------------------------------------
+# Try pretty printing the test results.
+# (*) example above was done manually
+#[root@ansibler]
+
+    jq '.' /tmp/test-result.json
+
+    >   parse error: Invalid numeric literal at line 1, column 5
+
+
+    sed '1d' /tmp/test-result.json | jq '.'
+
+    >   parse error: Invalid numeric literal at line 1, column 9
+
+
+
+
+# -----------------------------------------------------
+# -----------------------------------------------------
+
+    Second test run ..
+
+    changed: [localhost] => {
+        "changed": true,
+        "cmd": "python3 /tmp/run-test.py | tee /tmp/test-result.json",
+        "delta": "2:33:50.063424",
+        "end": "2022-01-21 15:54:20.780910",
+        "rc": 0,
+        "start": "2022-01-21 13:20:30.717486",
+        "stderr": "",
+        "stderr_lines": [],
+        "stdout": "Test completed after: 9229.76 seconds
+            {
+            'SetUp': {
+                'totaltime': '48.17',
+                'status': 'SLOW',
+                'msg': '',
+                'valid': 'TRUE'
+                },
+            'Mean_proper_motions_over_the_sky': {
+                'totaltime': '46.63',
+                'status': 'SUCCESS',
+                'msg': '',
+                'valid': 'TRUE'
+                },
+            'Source_counts_over_the_sky.json': {
+                'totaltime': '16.39',
+                'status': 'SUCCESS',
+                'msg': '',
+                'valid': 'TRUE'
+                },
+            'Good_astrometric_solutions_via_ML_Random_Forrest_classifier': {
+                'totaltime': '525.33',
+                'status': 'SLOW',
+                'msg': '',
+                'valid': 'TRUE'
+                },
+            'QC_cuts_dev.json': {
+                'totaltime': '4232.19',
+                'status': 'SUCCESS',
+                'msg': '',
+                'valid': 'TRUE'
+                },
+            'WD_detection_dev.json': {
+                'totaltime': '4361.05',
+                'status': 'SLOW',
+                'msg': '',
+                'valid': 'TRUE'
+                }
+            }
+        ....
+
+    #
+    # Are these results any goos ?
+    # What configuration were the original benchmarks made on ?
```

Comment:
The benchmark values that are there now were based on a run using the "cclake-large-06" deploy